doc/nrf/doc_build.rst

```diff
@@ -61,21 +62,33 @@ Complete the following steps to build the documentation output:
 
 1. Open a shell and enter the doc folder :file:`ncs/nrf/doc`.
 
-   * On Windows:
+   .. tabs::
 
-     a. Navigate to :file:`ncs/nrf`.
-     #. Hold shift and right-click on the :file:`doc` folder.
-        Select :guilabel:`Open command window here`.
+      .. group-tab:: Windows
 
-   * On Linux or macOS:
+         a. Navigate to :file:`ncs/nrf`.
+         #. Hold shift and right-click on the :file:`doc` folder.
+            Select :guilabel:`Open command window here`.
```

Comment:
There is still only the "Open PowerShell window here" option available.

Reply:
Unified to `command-line window` after discussion on Teams with @wiba-nordic and @Avalei .